SUMMARY.md

```diff
@@ -19,6 +19,30 @@
   * [Naming patterns](/html-css/naming-patterns.md)
   * [Forms](/html-css/forms.md)
   * [CSS processors](/html-css/css-processors.md)
+* [Java](/java)
```

Comment:
This should probably go towards the end, as it isn't a core module, we don't want the students confusing Java vs Javascript right at the beginning of the course